testdata/public/package/datasources-1.0.0/index.json

```diff
@@ -87,11 +147,84 @@
               "type": "password"
             }
           ],
-          "description": "Collecting metrics for nginx."
+          "description": "Collecting metrics for nginx.",
```

Comment:
I approved too fast.. 🙂 During our call earlier we also discussed the need for `dataset` property, i.e. this line in agent config spec: https://github.com/elastic/beats/blob/feature-ingest/x-pack/agent/docs/agent_configuration_example.yml#L50
I can't find it in this PR?

Reply:
I added the dataset in the most recent commit. Unfortunately it is not the full name yet as it depends on https://github.com/elastic/package-registry/pull/176 

I'm still debating on the name here but I assume if we change it from foo to bar, the change on your end should be pretty straightforward?

Reply:
yeah, doesn't matter what the property name is, as long as it's there